src/github/pullRequestOverview.ts

```diff
@@ -36,7 +37,8 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
 	protected static readonly _viewType: string = 'PullRequestOverview';
 
 	private _repositoryDefaultBranch: string;
-	private _existingReviewers: ReviewState[];
+	private _existingReviewers: ReviewState[] = [];
+	private _existingAssignees: AssigneeState[] = [];
```

Comment:
Actually, I think it might be possible to remove this cache completely. You should have access to `this._item.assignees`, which I think can be used instead